ush/templates/FV3LAM_wflow.xml

```diff
@@ -464,39 +503,28 @@ MODULES_RUN_TASK_FP script.
     <dependency>
       <and>
         <timedep><cyclestr offset="&START_TIME_CONVENTIONAL;">@Y@m@d@H@M00</cyclestr></timedep>
-        <taskdep task="&MAKE_LBCS_TN;"/>
         <taskdep task="&MAKE_ICS_TN;"/>
       </and>
     </dependency>
 
   </task>
-{%- endif %}
```

Comment:
Thanks for your great efforts!  With this PR, there is far less need to have the pair {% if not is_rtma %}   {%- endif %}. However can we keep this pair? I am sure it is the only rtma logic to turn off tasks now.